gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/test/java/io/gravitee/am/gateway/handler/root/resources/endpoint/mfa/MFAChallengeEndpointTest.java

```diff
@@ -99,6 +98,9 @@ public class MFAChallengeEndpointTest extends RxWebTestBase {
     private EmailService emailService;
     @Mock
     private AuthenticationFlowContextService authenticationFlowContextService;
+    @Mock
+    private AuditService auditService;
```

Comment:
todo: In this Test class could be nice to check that tha auditService is called with the right type of Event IMO.

Reply:
done, as suggested